acceptance/ui/features/steps/hosts_steps.rb

```diff
@@ -15,9 +27,38 @@
 end
 
 When /^I click the Add-Host button$/ do
-    clickAddHostsButton()
+    clickAddHostButton()
+end
+
+Then (/^the "Active" column should be sorted with active hosts on (top|the bottom)$/) do |order|
+  list = page.all("[ng-if$='host.active']")
```

Comment:
Did you try this with hosts that were NOT active?  I suspect it won't work. If you take a look at [view-hosts.html](../../../../web/ui/src/Hosts/view-hosts.html), you'll see that there are 2 separate `<div>`s for active vs inactive.  My guess is that selector expression here will only match the active ones.  Off the top of my head, I'm not sure how to fix this, though I'm sure with some experimentation there's a way to do it.  My Ruby knowledge is pretty slim, but maybe something like this:
```
   list = page.all("[ng-if$='host.active']")
   list += page.all("[ng-if$='!host.active']")
```
If this turns out to be hard to fix, don't get stuck on it. We can leave it as a known issue and circle back to fix it later.